HomeTask2905/CardAcc.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HomeTask2905
+{
+    public class CardAcc:BankAcc, InterestRate
+    {
+        public double Percent { get; set; }
+
+        public CardAcc(double balance, string owner)
+            :base(balance,owner)
+        {
+            
+        }
+
+        public void InterestAccrual()
+        {
+            this.Balance += this.Balance * (this.Percent/100);
+            CurrentBalance();
+        }
+
+        public void SettingPercent(double percent)
```

Comment:
The name of method is not a verb.
Also no need of this method since you have public property.